### DISCUSSION TITLE
Transform module availablity

### DISCUSSION THREAD

**Discussion by iamprayush** [apache/flink-cdc#3287] [Q&A] [answered]
Hello, I wanted to check what the current status of the transform module is. The documentation mentions that this feature will be available soon. Is there any estimated timeline to the development?

**Comment by yuxiqian** [comment, marked as answer]
Hi @iamprayush, PR #2937 which adds Transform feature has been merged and will be available in release 3.1. This version is still under ASF voting process, expected to be officially released in the following days, and corresponding docs will be added then.

**Reply by iamprayush** [reply]
Hi @yuxiqian, until the 3.1 version is released, for testing it, what would be the process of building my own jar and binaries that support transformations? Also, is there any documentation on the syntax of the transformation to get a sense of the capabilities?

**Reply by yuxiqian** [reply]
Hi @iamprayush, You may switch to `release-3.1` branch and compile a snapshot Flink CDC version to test it in advance. (Since package name has been changed in v3.1, you may need to compile a full release of `flink-cdc-dist` package and corresponding pipeline connectors.) PR #3308 adds transform-related documentations. I'll push it to be merged today.
